content/yaml-running-builds/scheduling.md

```diff
@@ -9,20 +9,15 @@ aliases:
   - /configuration/scheduling
 ---
 
-{{<notebox>}}
-**Note:** This feature is available for [teams](../teams/teams) only.
-{{</notebox>}}
-
 You can schedule Codemagic to automatically build your app on certain days and times. This way, your QA can start the day with the latest version of the app, or you can configure a workflow to run the long test suits at a convenient time and only run fast tests with builds triggered by commits to the repo.
 
 ## Setting up scheduled builds
 
-1. Open your project in Codemagic.
-2. On the right sidebar under **Scheduled builds**, click **Add new schedule**.
-3. Select the **Branch** and the **Workflow** to run.
-4. In the **Schedule for** field, select the days you want to run the build. 
-5. Specify the start time (UTC) of the build by selecting a value from the **At** field. Note that the build may be delayed up to 15 minutes during peak hours.
-6. Click **Add schedule** to save the schedule.
+1. Open your application in Codemagic.
```

Comment:
It's missing the info about clicking the Scheduled builds tab and Add new schedule button.